### ISSUE TITLE
Letters migration to Lighthouse endpoints

### ISSUE THREAD

**Issue by vhenry07** [department-of-veterans-affairs/va.gov-team#36345]
## Issue Description
Many of EVSS backend services are migrating to Lighthouse. We will need to work with EVSS to update our Letters endpoints. 

---
## Tasks
- [ ] Update and test new Lighthouse endpoints for the Letters application.

## Acceptance Criteria
- [ ] Endpoints updated and functioning properly. 
---

**Comment by cnemeth** [comment]
[See Slack discussion on Lighthouse readiness on this](https://dsva.slack.com/archives/CKUCFQAHJ/p1645117127623339)

**Comment by cnemeth** [comment]
##  Preliminary Determination of Tasks

1. Add new `Lighthouse::Letters` API, e.g., `vets-api/lib/lighthouse/letters` # => contingent on Lighhouse readiness
2. Update `app/controllers/v0/letters_controller.rb` to use new `Lighthouse::Letters` API
3. Update `modules/mobile/app/controllers/mobile/v0/letters_controller.rb` to use new `Lighthouse::Letters` API
4. Remove obsolete `Lighthouse::Letters` API # => `vets-api/lib/lighthouse/letters`

**Comment by NCHLiberty** [comment]
@cnemeth responded in slack but also about the feature toggle, that is a great idea and also, is it possible to build it so  only partial volume is shifted initially?

**Comment by cnemeth** [comment]
@NCHLiberty Yes, it is possible to split the traffic between EVSS and Lighthouse. The questions is, how do we decide on how we split the traffic between the two services?

**Comment by cnemeth** [comment]
@NCHLiberty Sorry, I don't really know who would the right product person to contact on this. Maybe @vhenry07 can help you with this.

**Comment by michelpmcdonald** [comment]
Ok, on the issue of directing some traffic to the LH letters endpoint and leaving some traffic still using the old endpoint, we have all that already via flipper and we've done that sort of thing before, so no new 'load balancer' has to be written for slow rolling out this new endpoint.
